tapir/core/tapir_email_base.py

```diff
@@ -17,9 +17,17 @@
 
 all_emails: Dict[str, Type[TapirEmailBase]] = {}
 
+enabled_by_default_options = Literal[True, False, "both"]
```

Comment:
What's the problem with using None instead of "both"?

Reply:
I think readability. For example, if I want both default and non-default values, `None` would read like I don't get None of it

Reply:
Even as an optional parameter? It's quite standard at least in Python.
It would look like `get_mails()` for all mails, `get_mails(mandatory=True)` with just one parameter, `get_mails(mandatory=False, enabled_by_default=True)` with both, isn't that fine?